### PR TITLE
[REF] pylint: Change number of message by name of message and add multiline

### PR DESCRIPTION
**Pull request by moylop260**
* Numbers2name
 Many times we need to check the pylint official documentation to get pylint-number information.
 But if we use the message-short-name is easiest identify it.
 By example:
  - `duplicated-key` is more explicit that `W0109`

* Multi-lines
 - To reduce conflicts of use same line if add new or remove old checks.
 - To identify a PR diff easiest.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Is there possibility to let as comment aside each error number?

**Comment by moylop260** [comment]
@pedrobaeza 
Good point
Let me try it

**Comment by moylop260** [comment]
@pedrobaeza 
Multilines config file don't support comment but I add fixed of other way.
Can you give me your opinion, please?

**Comment by pedrobaeza** [comment]
OK, that can serve. Why not ordering by number instead of by name?

**Comment by moylop260** [comment]
@pedrobaeza 
Added comment:
https://github.com/OCA/maintainer-quality-tools/pull/238#issue-103085045

File: travis/cfg/travis_run_pylint.cfg [hunk 1]
```diff
@@ -6,7 +6,39 @@ cache-size=500
...
+    file-ignored,
+    pointless-statement,
+    pointless-string-statement,
+    print-statement,
+    redundant-keyword-arg,
+    reimported,
+    relative-import,
+    return-in-init,
+    too-few-format-args,
+    unreachable,
```
Comment (dreispt) [inline travis/cfg/travis_run_pylint.cfg]: Is it allowed to add a comments at the end of each line? 
If so I suggest to add the message code at the end of each line and remove the comment block above.

Comment (moylop260) [inline travis/cfg/travis_run_pylint.cfg]: I tested with next cases:

```conf
enable=anomalous-backslash-in-string,  # comment here
    assignment-from-none,
```

```conf
enable=anomalous-backslash-in-string,
    # comment here
    assignment-from-none,
```
And don't work.

Comment (dreispt) [inline travis/cfg/travis_run_pylint.cfg]: Pity. No other option than leaving as it is, then. Thanks for trying.

**Comment by dreispt** [comment]
:+1:

**Comment by moylop260** [comment]
@dreispt 
Thank you!

@pedrobaeza 
Do you have another comment?

**Comment by pedrobaeza** [comment]
Nop, let's wait for another review or 5 days

**Comment by moylop260** [comment]
@pedrobaeza 
Ok, I got it
Thank you

**Comment by nhomar** [comment]
for me a :+1:

**Comment by moylop260** [comment]
Three approvals and pr of 6 days ago... then merging.

**Pull request merged**